crowbar_framework/lib/crowbar/upgrade.rb

```diff
@@ -52,6 +54,22 @@ def knife_files
           file_path = new_file
         end
 
+        if file.basename.to_s =~ /^bc-network-(.*)\.json$/
+          json = JSON.load(file.read)
+          # override with migrated attributes
+          json["attributes"] = SchemaMigration.migrated_attributes("network", json)
+          Tempfile.open("network") do |content|
+            content.write JSON.pretty_generate(json["attributes"])
```

Comment:
I was confused for one second because `content` is actually not the content, but a file object. Would it make sense to rename that variable?